packages/skills/weather_station_ledger/skill.yaml

```diff
@@ -17,7 +17,8 @@ shared_classes:
       class_name: Strategy
       args:
         price_per_row: 1
-        currency_pbk: 'FET'
+        currency_pbk: ['FET', 'ETH']
```

Comment:
this should be either `FET` or `ETH`, in this case `ETH` as `ledger_id` is `ethereum`